makelib/common.mk

```diff
@@ -105,13 +105,6 @@ endif
 # Set the host's arch.
 HOSTARCH := $(shell uname -m)
 
-# Set safe architectures for supported OSes (darwin and linux).
-# Apple Silicon binaries are not widely available yet
```

Comment:
@tnthornton I think this will at least break OLM bundle (https://github.com/upbound/olm-bundle/blob/f90948c2a3fb344f6b1d13b346b9ca1b62c89c9b/Makefile#L4) and potentially a few of the other tools. I wonder if we can either:
1. Update to compatible versions of each tool.
2. Say that you should set SAFEHOSTARCH manually if you are on Apple silicon and want to have a good time.